python-package/xgboost/sklearn.py

```diff
@@ -824,9 +860,18 @@ def apply(self, X, ntree_limit=0):
             ``[0; 2**(self.max_depth+1))``, possibly with gaps in the numbering.
         """
         test_dmatrix = DMatrix(X, missing=self.missing, nthread=self.n_jobs)
+        if iteration_range is None:
```

Comment:
This is a breaking change, to be consistent with the `predict` function.  Not sure if it's necessary.